api_tests/src/actor_test.ts

```diff
@@ -24,7 +24,7 @@ function nActorTest(
         const actors = await createActors(name, actorNames);
 
         try {
-            await timeout(60_000, testFn(actors));
+            await timeout(120_000, testFn(actors));
```

Comment:
It basically took this verbatim from Franck as "we need this to be higher for it to work on Mac" but that was quite some time ago and quite a few things changed since then.

I agree with you that we should investigate those test failures instead of randomly adding time here, esp. if this actually doesn't fix it properly :/

I hoped we could do this in a follow-up though :D